gobblin-modules/gobblin-azkaban/src/main/java/org/apache/gobblin/service/modules/orchestration/AzkabanClient.java

```diff
@@ -117,7 +117,14 @@ protected AzkabanClient(String username,
         .withWaitStrategy(WaitStrategies.exponentialWait(60, TimeUnit.SECONDS))
         .withStopStrategy(StopStrategies.stopAfterAttempt(3))
         .build();
-    this.sessionId = this.sessionManager.fetchSession();
+    try {
+      this.sessionId = this.sessionManager.fetchSession();
+    }
+    catch (Exception e) {
+      this.sessionCreationTime = -1;
+      log.error("Failed to fetch session in constructor due to " + e);
```

Comment:
Can you rewrite this to, `log.error("Failed to fetch session during initialization due to ", e)`
If you concat the exception with the log like this, it won't print out the stack trace, and under the hood it just transforms the exception itself to the object reference.